doc/docs/Python_Tutorials/Near_to_Far_Field_Spectra.md

```diff
@@ -662,7 +662,7 @@ if __name__ == "__main__":
 
 Note: in the case of a disc, the set of dipoles within the quantum well (QW) which spans a 2D surface only needs to be computed along a line. This means that the number of single-dipole simulations necessary for convergence is the same in cylindrical and 3D Cartesian coordinates.
 
-Note: for randomly polarized emission from the QW, each dipole requires computing the emission from the two orthogonal "in-plane" polarization states of $E_r$ and $E_\phi$ separately and averaging the Poynting flux in post processing. (The averaging is based on the principle that the emission from dipoles with different polarizations, whether the dipoles are positioned at the same or different locations, is incoherent.) In this example, only the $E_r$ polarization state is used.
+Note: for randomly polarized emission from the QW, each dipole requires computing the emission from the two orthogonal "in-plane" polarization states of $E_r$ and $E_\phi$ separately and averaging the Poynting flux in post processing. (The averaging is based on the principle that, in an isotropic medium, the emission from dipoles with different polarizations along the principal axes is incoherent. This is true whether the dipoles are positioned at the same or different locations within the QW.) In this example, only the $E_r$ polarization state is used.
```

Comment:
If it's isotropic, *every* set of orthogonal axes are principal axes (the eigenvalues of the polarization are degenerate).  So you don't need to mention principal axes.   

Note also that you're really only assuming that the QW polarizability *in plane* is isotropic.  You're assuming the QW has negligible polarizability out of plane, because you have only in-plane sources.

Reply:
Fixed.